src/Build/BackEnd/Components/Communications/NodeManager.cs

```diff
@@ -352,7 +349,7 @@ NodeConfiguration AcquiredNodeConfigurationFactory(NodeInfo nodeInfo)
 
         public IEnumerable<Process> GetProcesses()
         {
-            return _outOfProcNodeProvider.GetProcesses();
+            return _outOfProcNodeProvider?.GetProcesses() ?? new List<Process>(0);
```

Comment:
IEnumerable.Empty<Process>()

Reply:
It is dead path of code. _outOfProcNodeProvider is supposed to be always set in all codepaths calling this method. However, because of `nullable` we need to eliminate warning by `?`. I would actually recommend ` return _outOfProcNodeProvider?.GetProcesses()!`